src/main/resources/templates/instructor/add_instructor.html

```diff
@@ -37,17 +36,16 @@ <h1>Add New Instructor</h1>
     <table class="table table-striped">
        <thead>
           <tr>
-              <th>uid</th>
-              <th>username</th>
+              <th>UserID</th>
+              <th>Username</th>
               <th></th>
           </tr>
         </thead>
         <tbody>
-           <tr th:each="a: ${appUsers}"
-                   th:if="${a.IsInstructor}">
+           <tr th:each="a: ${appUsers}">
               <td th:text="${a.uid}"></td>
               <td th:text="${a.username}"></td>
-              <td th:if="${a.isInstructor}">
+              <td th:if="${true}">
```

Comment:
This might need to be changed since we have to make sure the instructor needs to be checked.